### ISSUE TITLE
One click link?

### ISSUE THREAD

**Issue by netmindz** [Pixelvation/Pixelvation#1]
Are you able to setup a link that people are able to order correctly just by clicking single link?

**Comment by embedded-creations** [comment]
I don't think so unless JLCPCB has improved their tools in the last year or so since I looked at them.  I'd like to start getting these manufactured at some point, it just hasn't been a priority recently.

Send me an email or PM on community.pixelmatix.com and I'll send you one
